tests/sdk/v2/driver/test_ce_runtime.py

```diff
@@ -1186,6 +1186,58 @@ def test_exception_handling(
             runtime.get_workflow_logs(workflow_run_id)
 
 
+class TestGetSystemLogs:
+    def test_happy_path(
+        self,
+        mocked_client: MagicMock,
+        runtime: _ce_runtime.CERuntime,
+        workflow_run_id: str,
+    ):
+        # Given
+        sys_logs = [
+            "<message sentinel 1>",
+            "<message sentinel 2>",
+            "<message sentinel 3>",
+        ]
+        mocked_client.get_system_logs.return_value = sys_logs
+
+        # When
+        logs = runtime.get_system_logs(workflow_run_id)
+
+        # Then
+        mocked_client.get_system_logs.assert_called_once_with(workflow_run_id)
+        assert logs == sys_logs
+
+    @pytest.mark.parametrize(
+        "exception, expected_exception",
+        [
+            (_exceptions.InvalidWorkflowRunID, exceptions.WorkflowRunNotFoundError),
+            (_exceptions.WorkflowRunNotFound, exceptions.WorkflowRunNotFoundError),
+            (_exceptions.InvalidTokenError, exceptions.UnauthorizedError),
+            (_exceptions.ForbiddenError, exceptions.UnauthorizedError),
+            (_exceptions.UnknownHTTPError, _exceptions.UnknownHTTPError),
+            # (
+            #     _exceptions.WorkflowRunLogsNotReadable,
+            #     exceptions.InvalidWorkflowRunLogsError,
+            # ),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Commented out code